app/src/main/java/org/fossasia/pslab/communication/sensors/SSD1306.java

```diff
@@ -0,0 +1,469 @@
+package org.fossasia.pslab.communication.sensors;
+
+import org.fossasia.pslab.communication.peripherals.I2C;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.concurrent.TimeUnit;
+
+import static java.lang.Math.abs;
+
+/**
+ * Created by akarshan on 4/20/17.
+
+ *  //  functions that are needed to be handled in ScienceLab.java
+        load("logo");
+        scroll("topright");
+        TimeUnit.MILLISECONDS.sleep(2800);
+        scroll("stop");
+
+ *      ScienceLab instance of i2c needs to be passed to SSD1306 class constructor
+ */
+
+public class SSD1306 {
+    public int ADDRESS = 0x3C;
+    public HashMap<String, ArrayList> params = new HashMap<>();
+    public int NUMPLOTS = 0;
+    public String[] PLOTNAMES = {""};
+    public String name = "OLED Display";
+
+    private int _width = 128;
+    private int WIDTH = 128;
```

Comment:
I left that to avoid confusion with the WIDTH. However, they are having different case but it felt that it should remain _width.